scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/SimulationFacadeTest.java

```diff
@@ -353,4 +356,21 @@ public void testActivityTypeWithResourceConstraint() throws SchedulingInterrupte
     assertTrue(TestUtility.containsExactlyActivity(plan, act2));
     assertTrue(TestUtility.doesNotContainActivity(plan, act1));
   }
+
+  @Test
+  public void testIdMapOnCachedPlan() throws SchedulingInterruptedException, SimulationFacade.SimulationException {
```

Comment:
I'm not sure I follow how this test shows that the plan was not resimulated, but instead a cache was loaded with updated ids. Could you please explain it?

Reply:
Ah, I see. I misread the bug and thought there was an issue in the caching, not the returned ids. You're right, just checking that the new ids are applied is correct for the test, since caching is tested elsewhere (ie `testOptionalSimulationAfterGoal_staleResources`)